### DISCUSSION TITLE
filtering sources based on bitrate?

### DISCUSSION THREAD

**Discussion by overtoneblue** [itsToggle/plex_debrid#122] [Q&A]
Hi, first I wanted to say thanks a lot for contributing to this open source project and maintaining it. I was wondering if there was a way to filter query results based on bitrate? I know there are some parameters for min/max file size, however this will obviously vary quite a bit depending on tv shows or movies. For example, if i'm searching for a movie, and I sort by size ascending/descending, i'll either grab a file around 1gb or 60gb in some cases. While most of the time specifying resolution and some other options may help a bit, I wanted to see if there was a more consistent way to achieve a bitrate range of 4000kbps-10000kbps as an example. 

Note: sorry for making two discussions back to back, hopefully some useful information can be gleaned from my questions. As a sidenote, a community chat of some sort may be useful as the project grows (aside from github discussions), especially due to the open source nature of the script, it would be interesting to see various ideas floating around with less barrier to entry than posting here. Thanks again

**Comment by itsToggle** [comment]
Sounds like a great idea, I will start a discord server sometime next week :)
There sadly isn't really a way to sort by Bitrate. The only useful attributes available to jackett (and therefore plex_debrid) are "size", "title" and "seeders". Releases do include a lot of information in the title, like resolution and video/audio codec, but Bitrate is usually not one of them.
It's possible to find out the Bitrate of a given release by checking the corresponding nfo file, but implementing a check like that into plex_debrid would increase scraping times a whole lot.

**Reply by overtoneblue** [reply]
yeah I figured something like that would increase scraping times by quite a bit, I'll still try and include some relevant parameters and If I happen to stumble into a decent release sorting config I will come back here and post in this discussion. Discord server would be great, i'll make sure to keep my eye on the github page.
